### DISCUSSION TITLE
Trick to add linkable content in automatic `listing`

### DISCUSSION THREAD

**Discussion by jlperla** [quarto-dev/quarto-cli#11720] [Q&A] [answered]
### Description

I have a number of files, e.g. `slides/intro.qmd` which generate a `slides/intro.html` in revealjs and I create a `slides/intro.pdf` in a separate toolchain.

I am generating a listing for all of my `slides/*.qmd` with
```yaml
---
title: "Course Overview" 
listing:
  contents: slides
---
```

I love the look of the `listing` but really want to have an ability to link to both the genreated html and my associated pdf.  Is there any "hack" to get a link to these into the listing (either default or grid)?  I tried to put in metadata in my files, such as a `slides/intro.pdf` with
```yaml
---
title: "Intro Slides"
description: "Some text and a link: [PDF](slides/intro.pdf)"
---
MY CONTENT
```

In that case, the PDF link shows in the listing, but is not a clickable link.  Any ideas or hacks which can get it associatd with the listing?

**Comment by mcanouil** [comment, marked as answer]
Build your own listing (https://quarto.org/docs/websites/website-listings-custom.html) and/or use format-links (https://quarto.org/docs/output-formats/html-multi-format.html) and/or other-links (https://quarto.org/docs/output-formats/html-basics.html#code-links-and-other-links).

**Reply by mcanouil** [reply]
See Andrew Heiss custom listing for his research at https://www.andrewheiss.com/research/
There are plenty of examples.

**Reply by jlperla** [reply]
Thanks.  I was looking at your suggestion of `https://quarto.org/docs/output-formats/html-multi-format.html`.  Will those get put into listings automatically?  I put `format-links` on my revealjs doc yaml setup in a few different ways  but it didn't show in the listings.

**Reply by mcanouil** [reply]
no, as documented it adds the links of the other formats in the `html` format.
